torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -32,6 +32,40 @@ graph(%a_quant, %packed_params, %r_scale, %r_zero_point, %r_dtype, %stride, %pad
         %r_quant = aten::quantize_per_tensor(%r, %r_scale, %r_zero_point, %r_dtype)
         return (%r_quant) )";
 
+  // aten::conv1d - aten::relu
+  std::string conv1d_relu = R"(
+graph(%a_quant, %packed_params, %r_scale, %r_zero_point, %r_dtype, %stride, %padding, %dilation, %groups):
+        %a_dequant = aten::dequantize(%a_quant)
```

Comment:
For my understanding, where is the pack op inserted for the weights? I was expecting to see a wt->q->pack->unpack->dq->Conv

Reply:
yes, it's inserted before, here we just fuse `unpack - dq - conv - q` to quantized::conv2d